### DISCUSSION TITLE
OpenTelemetry SDK

### DISCUSSION THREAD

**Discussion by plunkettscott** [spatie/laravel-open-telemetry#3] [Q&A] [answered]
Is there any reason you're not just building a package that configures the OpenTelemetry SDK and wraps Laravel-specific features? That alone would be a HUGE help to people who already have other services instrumented with OpenTelemetry in other languages.

I feel like building your own implementation of the spec will add a lot of unnecessary work to this package. There's an official "example" of OpenTelementry in Laravel [here](https://github.com/open-telemetry/opentelemetry-php/blob/main/docs/laravel-quickstart.md).

**Comment by freekmurze** [comment, marked as answer]
I've done this for a few other packages, but with mixed result. The office SDK feels alpha and a lot of breaking changes can still be made. I don't want to be hit by those changes.

Our package is kinda simple, and it actually uses some small parts from the SDK that were not trivial to implement. I feel that, as a maintainer, this gives me the best of both worlds.
